### PR TITLE
ACRS-20 : Changed external ingress in prod to internal ingress and added staging env

### PR DESCRIPTION
**Pull request by DivineTaminangHO**
## What? 

Changing external ingress for prod env in acrs to internal ingress
Also added the staging environment in the pipeline

## Why? 

This ingress is used for internal communication between pods and services and not from the browser
Staging is added to be able to deploy to staging environment 

## How? 
For prod in ingress.yaml, the external ingresses have been changed to internal ingress 
## Check list

- [ x] I have reviewed my own pull request for linting issues (e.g. adding new lines)
- [ ] I have written tests (if relevant)
- [ x] I have created a JIRA number for my branch
- [ x] I have created a JIRA number for my commit
- [x ] I have followed the chris beams method for my commit https://cbea.ms/git-commit/
here is an [example commit](https://github.com/UKHomeOfficeForms/hof/commit/810959f391187c7c4af6db262bcd143b50093a6e)
- [x ] Ensure drone builds are green especially tests
- [x ] I will squash the commits before merging

### CODE REVIEW

**Comment by adityababumallisettiHO** [comment]
@DivineTaminangHO We need to replace External Ingress annotations to Internal Ingress ingress annotations in deploy.sh

**Review by TemitopeAyokuHO — APPROVED** [review]

**Comment by TemitopeAyokuHO** [comment]
LGTM

File: kube/hof-rds-api/ingress.yml [hunk 1]
```diff
@@ -23,13 +23,13 @@ spec:
         - data-service-{{ .DRONE_SOURCE_BRANCH }}.branch.sas-notprod.homeoffice.gov.uk
       {{ end }}
       {{ if eq .KUBE_NAMESPACE .BRANCH_ENV }}
-      secretName: branch-tls-external
-      {{ else }}
+      secretName: branch-tls-internal
+      {{ else }}
```
Comment (jamiecarterHO) [inline kube/hof-rds-api/ingress.yml:27]: Space added on end of line here...probably ok but hopefully it won't mess with the yaml

**Review by jamiecarterHO — CHANGES_REQUESTED** [review]
Mostly looking good! Couple of questions...

**Review by DivineTaminangHO — COMMENTED** [review]
changed external ingress to internal

**Review by DivineTaminangHO — COMMENTED** [review]
removed spaces

**Review by jamiecarterHO — APPROVED** [review]
Thanks for updates, looking good.

**Pull request merged**